src/MultiInput/MultiInput.test.js

```diff
@@ -38,62 +37,50 @@ describe('<MultiInput />', () => {
 
     let wrapper;
 
-    const getListStatus = (bIsShown) => {
+    const getListStatus = () => {
         const combobox = wrapper.find(
-            `div.fd-combobox-control[aria-expanded=${bIsShown}]`
+            'div.fd-popover__control'
```

Comment:
We should probably reinstate the `aria-expanded` attribute from the selector since it is being emitted.